devtools/maven/src/main/java/io/quarkus/maven/CreateProjectMojo.java

```diff
@@ -144,6 +160,26 @@ public void execute() throws MojoExecutionException {
         }
     }
 
+    private void createMavenWrapper() {
+        try {
+            executeMojo(
+                    plugin(
+                            groupId("io.takari"),
+                            artifactId("maven"),
+                            version(MojoUtils.getMavenWrapperVersion())),
+                    goal("wrapper"),
+                    configuration(
+                            element(name("maven"), MojoUtils.getProposedMavenVersion())),
+                    executionEnvironment(
+                            project,
+                            session,
+                            pluginManager));
+        } catch (Exception e) {
+            // no reason to fail if the wrapper could not be created
+            getLog().debug("Unable to create Maven Wrapper");
```

Comment:
So this bugs me a bit: if at some point it gets broken, our instructions will be incorrect as the `./mvnw` wrapper won't be there.

Wondering if we should throw an error?

Reply:
I see your point, but I think that preventing the users from creating the project just because the wrapper is missing will be more of a problem than (slightly) incorrect instructions. I am willing to bet that most users will understand that Maven wrapper is missing once they do `./mvnw ...` and the generic OS error message.

WDYT?

Reply:
Would like me to throw the exception after all @gsmet?

Reply:
Let's find a compromise and log an error instead of debug.

Reply:
I would also change the error message to: `Unable to install the Maven wrapper (./mvnw) in the project`.

Reply:
Sure, on it :)

Reply:
Fixed